configuration/rules-dsl.md

```diff
@@ -801,7 +801,8 @@ Caveat: Please note the semicolon after the return statement which terminates th
 
 ### Concurrency Guard
 
-If a rule triggers on UI events it may be necessary to guard against concurrency.
+If a rule is explicitly run from another script, rule, MainUI widget, instead of a trigger, the rule can be started before the current execution has ended.
```

Comment:
```suggestion
If a rule is explicitly run from another script, rule, a Main UI widget, instead of a trigger, the rule can be started before the current execution has ended.
```